hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/ExternalCondition.java

```diff
@@ -37,11 +37,11 @@
     private static final long serialVersionUID = 1L;
 
     /**
-     * An identifier assigned by the external system to identify this condition as being handled by that
-     * external system. It should be unique enough such that externalSystemIds are unique.
+     * An identifier assigned by the external alerter to identify this condition as being handled by that
+     * alerter. It should be unique enough such that external AlerterIds are unique.
      */
     @JsonInclude(Include.NON_NULL)
-    private String systemId;
+    private String alerterId;
```

Comment:
Purely for clarity, it unifies the terminology of "External Alerter" and how that Alerter identifies itself  when defining its conditions.